src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -115,6 +115,10 @@
 	(appendStringInfo(str, " :" CppAsString(fldname) " "), \
 	 _outBitmapset(str, node->fldname))
 
+#define WRITE_CUSTOM_FIELD(fldname, fldvalue) \
+	(appendStringInfo(str, " :" CppAsString(fldname) " "), \
+	appendStringInfoString(str, (fldvalue)))
+
```

Comment:
This felt hacky, but was the only way I could think of to have somewhat decent of a output for `TableDDLCommandFunction`. This will mostly be useful for debugging.

Am open to a better way if someone has an idea

Reply:
seems fine and better than not having it